glide.yaml

```diff
@@ -1,6 +1,6 @@
 package: kubevirt.io/containerized-data-importer
 import:
-- package: github.com/golang/glog
+- package: k8s.io/klog
```

Comment:
Don't forget to do
```bash
glide cc && glide up -v
```
That will update all the required dependencies. That will add a ton of files, which is fine.